front/src/applications/stdcmV2/components/StdcmConsist.tsx

```diff
@@ -41,24 +41,9 @@ const StdcmConsist = ({ setCurrentSimulationInputs, disabled = false }: StdcmCon
   const { rollingStock } = useStoreDataForRollingStockSelector();
 
   const { filters, searchRollingStock, searchRollingStockById, filteredRollingStockList } =
-    useFilterRollingStock();
-
-  const getLabel = (rs: LightRollingStockWithLiveries) => {
-    let res = '';
-    const { metadata, name } = rs;
-
-    if (metadata) {
-      const series = metadata.series ?? metadata.reference;
-      const subseries =
-        metadata.series && metadata.series !== metadata.subseries
-          ? metadata.subseries
-          : metadata.detail;
-      if (series) res += series;
-      if (subseries) res += series ? ` (${subseries})` : subseries;
-    }
-    if (name) res += ` -- ${name}`;
-    return res;
-  };
+    useFilterRollingStock({ isStdcm: true });
+
+  const getLabel = (rs: LightRollingStockWithLiveries) => rs.name;
```

Comment:
Is it "PO validated" that we now only want to see the name of the rolling stock in the suggestions ?

Reply:
It is something asked by Alex specifically yes, but I can keep it for a later PR :+1: